style.css

```diff
@@ -350,6 +762,114 @@ body {
   .range__link {
     font-size: 12px;
   }
+
+  .order {
+    padding: 8vw 5vw 0 6.5vw;
+  }
+
+  .order-images {
+    margin-right: 7vw;
+  }
+
+  .order-image__first {
+    margin-top: 110px;
+    height: 55%;
+    width: 70%;
+  }
+
+  .order-image__second {
+    height: 90%;
+  	width: 95%;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5841754)